asreview/models/classifiers/nn_2_layer.py

```diff
@@ -183,6 +191,11 @@ def _create_dense_nn_model(vector_size=40,
     _check_tensorflow()
 
     def model_wrapper():
+
+        if isfile('model.h5'):
+            model = load_model("model.h5")
+            return model
```

Comment:
This is risky because multiple simulations can be performed from the same folder. That might result in conflicts.